packages/fx-core/src/core/middleware/utils/migrationContext.ts

```diff
@@ -106,13 +106,21 @@ export class MigrationContext {
   }
 
   async cleanTeamsfx(): Promise<void> {
-    await fs.remove(path.join(this.projectPath, teamsfxFolder));
+    await this.fsRemove(teamsfxFolder);
+  }
+
+  async removeFxV2(): Promise<void> {
+    await this.fsRemove(V2TeamsfxFolder);
   }
 
   async fsPathExists(_path: string): Promise<boolean> {
     return await fs.pathExists(path.join(this.projectPath, _path));
   }
 
+  async fsRemove(_path: string): Promise<void> {
+    return await fs.remove(path.join(this.projectPath, _path));
```

Comment:
it's better to check the folder exist, then remove

Reply:
The remove method has existing check inside.